install/scripts/php-conf-fix.sh

```diff
@@ -6,7 +6,24 @@
 #
 echo 'Automated php.ini configuration adjustments'
 echo
-phpIni=/etc/php5/apache2/php.ini
+if [ -f /etc/debian_version ]
```

Comment:
I would say the default case is the debain case which might match most other distros (as shown in the code snipped above).